spec/features/admin/configuration/dynamic_content_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+
+# Defines the test link string
+TEST_LINK = "ofn test link"
+
+# Defines the test href string. This one should contain all non-encoding characters, for a URL
+TEST_HREF = ":/?#@!$&'()*+,;=0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+
+describe 'add dynamic content to homepage' do
+	include AuthenticationHelper
+
+	before(:each) do
+		login_as_admin_and_visit spree.edit_admin_general_settings_path
+	end 
+	
+		
+	context 'edit the Content section', js: true do	
+		before(:each) do
+			click_link ("Content")
+		end 
+
+
+		# makes sure we are on the right page
+		scenario 'while on the Content section' do
```

Comment:
`#scenario` and `#it` are aliases in Rpsec, but we tend to prefer `#it` (I think `#scenario` is more of an Rspec 2 thing?).

For reference, `#describe` and `#context` are are the same code-wise, but we usually use `#describe` for the test description and `#context` if there are multiple contexts, eg:

```ruby
describe "viewing order cycles" do
  context "as an admin" do
    it "shows all order cycles" do
      ...
    end
  end 
  context "as an enterprise user"
    it "shows the user's order cycles" do
      ...
    end
  end
end
``` 

Also we tend to use `#it` to describe the behavior being tested in the example, so something like:

```ruby
context "editing the content section" do
  it "shows the footer content" do
    expect(page).to have_content footer_content
  end
end
```

I'm not sure we've documented this clearly...

Reply:
Thanks @Matt-Yorkley ,
I was indeed unsure which alias should I use, thanks for this. I'll consider this in future specs.
Would it make sense to refactor older specs into this style, ex. replacing scenario -> it ?